priv/gateway_rs/settings.toml

```diff
@@ -1,2 +1,8 @@
 ## api listening port override
 api = 4468
+
+[poc]
+entropy_uri = "https://entropy.helium.io/entropy"
+ingest_uri = "http://mainnet-pociot.helium.io:9080"
+beacon_interval = 600
```

Comment:
beacon_interval seems to be the only setting you actually need here. Ingest token is not needed on the lora network ingestor and the two URIs match the default values. 
you could set this as an environment variable for the gateway in miner_gateway_port as `GW_POC_BEACON_INTERVAL=600` if you prefer but I thought we had to rename that to just `interval`?